reacnetgenerator/_mergeiso.py

```diff
@@ -0,0 +1,44 @@
+from tqdm import tqdm
+import numpy as np
+
+from .utils import bytestolist, listtobytes,  SharedRNGData
+
+
+class _mergeISO(SharedRNGData):
+    def __init__(self, rng):
+        SharedRNGData.__init__(self, rng, ['miso', 'moleculetempfilename'],['temp1it'])
+
+    def merge(self):
+        if self.miso > 0:
+            self._mergeISO()
+        self.returnkeys()
+
+    def _mergeISO(self):
+        with open(self.moleculetempfilename, 'rb') as ft,open('/data/home/cx/test/reac/test.dat', 'wb') as ft2:
```

Comment:
ft2 is for debugging? The path is not correct.

Reply:
yes, I forget to remove this.